db/schema.rb

```diff
@@ -21,4 +21,15 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "tasks", force: :cascade do |t|
+    t.text "todo"
+    t.datetime "due"
+    t.text "notes"
+    t.bigint "category_id", null: false
```

Comment:
By default in rails, when you set a reference type, the association is required.
With this, when you create a task, a category is required to be associated. 
Refer to #14 for suggested fix 😃 
Ref: [Active Record reference](https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)